htdocs/takepos/admin/setup.php

```diff
@@ -41,19 +41,30 @@
 
 $langs->loadLangs(array("admin", "cashdesk"));
 
+global $db;
+
+$sql="SELECT code, libelle FROM ".MAIN_DB_PREFIX."c_paiement WHERE active=1 ORDER BY libelle";
+$resql = $db->query($sql);
```

Comment:
@dbeniamine 
for compatibility with multicompany : 

```
$sql = "SELECT code, libelle FROM ".MAIN_DB_PREFIX."c_paiement";
$sql.= " WHERE entity IN (".getEntity('c_paiement').")";
$sql.= " AND active = 1";
$sql.= " ORDER BY libelle";
```

Reply:
done in 32303bbe4827cba26e8b57b5d17844766a1fc2c9